obspy/io/rg16/tests/test_read_rg16.py

```diff
@@ -87,6 +89,14 @@ def test_starttime_endtime_option(self):
         for tr, tr1 in zip(st, st1):
             self.assertEqual(tr.stats.starttime, tr1.stats.starttime)
             self.assertEqual(tr.stats.endtime, tr1.stats.endtime)
+        
+        # test when starttime and endtime are comprised in a data blocket.
+        st2 = rc._read_rg16(THREE_CHAN_FCNT, starttime=t3, endtime=t4)
```

Comment:
If I understand the aim of the PR correctly you want the inside timestamps here, so by your current naming `starttime=t1, endtime=t4`. Did you get confused by your own variable naming? ;-P

Reply:
Also please rename in comment from `blocket` to maybe `packet` or `block`. "Blocket" sounds too much like MiniSEED blockettes.

Reply:
The file three_chans_six_trace.fcnt contains the data of three channels. Each channel owns 2 data block. The first data block starts at 16:00:00.380000 and ends at 16:00:30.378000, the second data block starts at 16:00:30.380000 and ends at 16:01:00.378000. If a user wants to retrieve data between 16:00:45 and 16:00:55, the function _read_rg16 will return an empty stream. That's why i added an if condition in core.py to take into account this case.

Reply:
I understand the problem, I was just curious if you used the starttime and endtime as intended here, but seems OK then. You should maybe add this information about the start/endtimes of the two data packets in each test file as a comment in the test, because the intention/aim of the test is not clear from the test code as is.